helpers/SendMail.js

```diff
@@ -0,0 +1,26 @@
+const nodemailer = require('nodemailer');
+
+const sendMail = (email, message) => {
+  const smtpTransport = nodemailer.createTransport({
+    service: 'gmail',
```

Comment:
this might not be a good thing currently as we are in the test state. 
i suggest that you look up [mailtrap](https://mailtrap.io).